installer/windows/build_installer.py

```diff
@@ -332,7 +332,7 @@ def create_wxs_file():
         <CustomAction
             Id="ConfigureAcrylic"
             Directory="INSTALLDIR"
-            ExeCommand="cmd /c &quot;[INSTALLDIR]{ACRYLIC_CONFIG_SCRIPT}&quot; >> {INSTALLER_LOGFILE} 2>&amp;1"
+            ExeCommand="powershell.exe -Command &quot; &amp; '[INSTALLDIR]{ACRYLIC_CONFIG_SCRIPT}' '[AcrylicDNSProxy]' &quot; >> $env:Temp/massastation_installer.log 2>&amp;1"
```

Comment:
Since the command is ran in powershell, we need to use `$env:Temp/...` instead of `%Temp%/...`
Sooo, the variable is still used everywhere else

The only possible issue is if we decide to change the installer logs path

Reply:
We could add some variables to handle that case such as:

```
INSTALLER_LOG_FILENAME="massastation_installer.log"
CMD_LOG_FILEPATH="%Temp%"+INSTALLER_LOG_FILENAME
POWERSHELL_LOG_FILEPATH="$env:Temp"+INSTALLER_LOG_FILENAME
```

That would ensure that the file is always the same between PS and CMD (unless we update only one of `{CMD/PS}_LOG_FILEPATH`)

Another way would be to define the TMP dir using WIX, so we could use the ref to the directory such as for the `[INSTALLDIR]`, and so, we would avoid issue such as a variable that wasn't updated

IMO, the WIX solution is safer, but will take more time to do than just adding some variables.
Maybe we can go with the variables and improve this using WIX once we have more time to do so ?

Reply:
okay, it acceptable like this. Can you create a follow up issue, we will prioritize it for later.

Reply:
Issue created here: https://github.com/massalabs/station/issues/1105